internal/wasm/jit/jit_amd64_test.go

```diff
@@ -345,7 +345,6 @@ func TestAmd64Compiler_compileBrTable(t *testing.T) {
 		require.NoError(t, err)
 
 		// Run codes
-		env := newJITEnvironment()
```

Comment:
bug

Reply:
I don't believe this is the cause of flakiness (as jitEnvironment has nothing to do with the code in this case) but at least this is inconsistent

Reply:
inserting `.Parallel()` under `t.Run(fmt.Sprintf("index_register=%d,tmpRegister=%d", indexReg, tmpReg)` causes similar segfalult so this must be something to do race

Reply:
I was wondering if there was a way to pre-initialize things. In java sometimes we eagerly resolve a reference to ensure no late pointers at runtime (ex static inner types we end up having to do this sometimes)